packages/cli/src/index.ts

```diff
@@ -493,8 +470,46 @@ yargs
           choices: ['warn', 'error', 'off'] as ReadonlyArray<RuleSeverity>,
           default: 'warn' as RuleSeverity,
         },
+        format: {
+          hidden: true,
+          deprecated: true,
+        },
+        lint: {
```

Comment:
Need to keep those options here if we want notify users, otherwise strict mode will force `help` display.